src/main/java/com/google/gcloud/storage/Blob.java

```diff
@@ -157,12 +157,12 @@ public Builder acl(List<Acl> acl) {
       return this;
     }
 
-    public Builder owner(Acl.Entity owner) {
+    Builder owner(Acl.Entity owner) {
```

Comment:
I don't see a problem with the Builder pattern I have there. The problem was that I exposed in the Builder setters that should only be used by the service client.

I think there various reasons to hide the underlying proto (even without adding functionality on top)
among them are:
(1) it is actually not a protobuf  but an Apiary code that is subject to change (e.g. when we move to gRPC).
(2) the hand-crafter layer is Immutable and only contain the relevant settings to the user.
(3) It allows us to simply when possible and eliminate unnecessary fields (or fields that are deprecated).